app/scripts/views/generator-view.js

```diff
@@ -103,17 +123,22 @@ const GeneratorView = Backbone.View.extend({
 
     generate: function() {
         this.password = PasswordGenerator.generate(this.gen);
-        this.resultEl.text(this.password);
+        this.showPassword();
         const isLong = this.password.length > 32;
         this.resultEl.toggleClass('gen__result--long-pass', isLong);
     },
 
+    hideChange: function(e) {
+        this.hide = e.target.checked;
+        AppSettingsModel.instance.unset('generatorHide', { silent: true });
```

Comment:
Looks like it can be removed because we set it later.

Reply:
Marking this resolved because I think it's the same as https://github.com/keeweb/keeweb/pull/1223#discussion_r312105527